website/docs/cli/commands/test/index.mdx

```diff
@@ -0,0 +1,368 @@
+
+<Tabs>
+    <TabItem value={"test"} label={"main.tftest.hcl"} default>
+        <CodeBlock language={"hcl"}>{ExpectFailureResourcesTest}</CodeBlock>
+    </TabItem>
+    <TabItem value={"main"} label={"main.tf"}>
+        <CodeBlock language={"hcl"}>{ExpectFailureResourcesMain}</CodeBlock>
+    </TabItem>
+</Tabs>
+
+### The `run.command` setting and the `run.plan_options` block
+
+By default, `tofu test` uses `tofu apply` to create real infrastructure. In some cases, for example if the real
+infrastructure is very expensive or impossible to run for testing purposes, it can be useful to only run `tofu plan`
+instead. You can use the `command = plan` setting to perform a plan instead of an apply. The following example tests if
+the variable is correctly passed to the `docker_image` resource without actually applying the plan:
+
+<Tabs>
+    <TabItem value={"test"} label={"main.tftest.hcl"} default>
+        <CodeBlock language={"hcl"}>{PlanTest}</CodeBlock>
+    </TabItem>
+    <TabItem value={"main"} label={"main.tf"}>
+        <CodeBlock language={"hcl"}>{PlanMain}</CodeBlock>
+    </TabItem>
+</Tabs>
+
+Regardless of the `command` setting, you can use the `plan_options` block to specify the following additional options for both modes:
+
+| Name    | Description                                                                                                                                               |
+|:--------|:----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| mode    | Change this option from `normal` (default) to `refresh-only` in order to only refresh the local state from the remote infrastructure.                     |
+| refresh | Set this option to `false` to disable checking for external changes in relation to the state file. Similar to `tofu plan -refresh=false`.                 |
+| replace | Force replacing the specified list of resources, such as `[docker_image.build]` in the above example. Similar to `tofu plan -replace=docker_image.build`. |
+| target  | Limit planning to the specified list of modules or resources. Similar to `tofu plan -target=docker_image.build`.                                          |
+
+:::tip Tip
+
+You can use these options in conjunction with [provider overrides](#the-providers-block) to realize fully offline tests. See the
+[Providers section below](#the-providers-block) for an example.
+
+:::
+
+### The `providers` block
+
+In some cases you may want to override provider settings for test runs. You can use the `provider` blocks outside of
+`run` block to provide additional configuration options for providers, such as credentials for a test account.
+
+```hcl
+provider "aws" {
+  // Add additional settings here
+}
+```
+
+This feature can also realize partially or fully offline tests if the provider supports it. The following example
```

Comment:
I don't think the word `realize` makes sense here.. not sure what to replace it with though :/

Reply:
"facilitate"? Sounds a bit pretentious though.

Reply:
How about "enable"?

Reply:
what about `conduct`? 

```suggestion
This feature can also conduct partially or fully offline tests if the provider supports it. The following example
```

or `carry out`

```suggestion
This feature can also carry out partially or fully offline tests if the provider supports it. The following example
```